src/styles/media.tsx

```diff
@@ -1,3 +1,7 @@
-export const mediaQuery = (minWidth: number) => `
+export const mediaQuery = (minWidth = 640) => `
   @media (min-width: ${minWidth}px)
 `;
+
+export const calcSize = (size: number) => `
+  ${size / 3.75}vw
+`;
```

Comment:
```suggestion
export const pxToVw = (px: number) => `
  ${px / 3.75}vw
`;
```

혹시 요런 느낌의 네이밍은 어떠실까유??? 그러면 지형님의 코드를 처음 보는 사람도 이 함수의 역할을 더 빠르게 파악할 수 있을 것 같아요 ㅎㅎ